learn/wavemaker-release-notes/v11-8-0.md

```diff
@@ -4,75 +4,81 @@ id: "v11-8-0"
 sidebar_label: "v11.8.0 - latest"
 ---
 
-WaveMaker release 11.8.0, includes significant upgrades like Angular 17 and Expo 50 along with Artifact sharing between WaveMaker Enterprise (WME) setups and  Single Page Application (SPA) behaviour as new features. In React Native,  new CSS style classes in Slider widget, setting range in Slider widget, Secure Sockets Layer (SSL) certificate pinning, navigation in Wizard widget, and Wheel date picker are some of the ehancements along with many bug fixes.
```

Comment:
Expo 50 **for React Native**
Artifact sharing **across** WaveMaker Enterprise (WME) **environments.**